# Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of
15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.


    Answer

To test the hypothesis, we need to perform a hypothesis test using a t-test.

Null Hypothesis: The raw cornstarch diet has no effect on blood glucose levels.


Alternative Hypothesis: The raw cornstarch diet has a positive effect on blood glucose levels.

We can use a one-sample t-test since we are comparing the mean glucose level of the sample of 36 patients to the population mean.











In [4]:
from scipy.stats import norm
import numpy as np
import math
import scipy.stats as stats
from scipy.stats import chi2_contingency

In [7]:
#The t-statistic can be calculated as follows:
import numpy as np
alpha=0.05
#t = (sample mean - population mean) / (standard deviation / sqrt(sample size))
t = (108 - 100) / (15 / np.sqrt(36))
t

stats.t.ppf((alpha/2),df=35)

-2.030107928250343

Using a t-distribution table with 35 degrees of freedom (sample size - 1), we find that the p-value for a two-tailed test is less than 0.001.

Since the p-value is less than the significance level of 0.05, we reject the null hypothesis and conclude that the raw cornstarch diet has a statistically significant positive effect on blood glucose levels.

# Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

    Ans 

To solve this problem, we need to calculate the sampling distribution of the difference in sample proportions between the two states, assuming that the null hypothesis of equal proportions is true. We can use the Central Limit Theorem to approximate the sampling distribution as a normal distribution, with mean equal to the difference in population proportions and standard deviation equal to the square root of the sum of the variances of the two sample proportions.

The probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is equivalent to the probability that the difference in sample proportions is greater than zero. We can calculate this probability using the standard normal distribution.

In [21]:
# Population proportions
p1 = 0.52
p2 = 0.47

# Sample sizes
n1 = 100
n2 = 100

# Difference in sample proportions
p_diff = p2 - p1

# Standard deviation of sampling distribution
se = np.sqrt((p1*(1-p1))/n1 + (p2*(1-p2))/n2)

# Z-score
z = p_diff / se

# Probability of z-score
prob = 1 - stats.norm.cdf(z)

print("The probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is:", round(prob, 4))


The probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is: 0.7605


# Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

To determine how well you scored on the test compared to the average test taker, we can calculate your z-score using the formula:


z = (x - μ) / σ


where x is your SAT score, μ is the mean SAT score, and σ is the standard deviation of the SAT scores.








In [13]:
#Substituting the given values, we get:
z = (1100 - 1026) / 209


0.35406698564593303

This means that your SAT score is 0.3541 standard deviations above the mean SAT score.


To interpret this result, we can use a standard normal distribution table or calculator to find the proportion of the population that falls below a z-score of 0.3541.



Using the table, we find that the proportion is 0.6368, which is approximately 63.68%.



Therefore, you scored better than approximately 63.68% of the test takers who took the SAT.


In [ ]:
#Intern of Pranathi
#Student of DataTrained - Saurav
#Date - 28-03-2023
#Time -> 17:37